config/wdio/wdio.conf.js

```diff
@@ -0,0 +1,56 @@
+const localIP = require('ip');
+
+const AxeService = require('../../lib/wdio/services').Axe;
+const TerraService = require('../../lib/wdio/services').Terra;
+const SeleniumDockerService = require('../../lib/wdio/services').SeleniumDocker;
+const visualRegressionConfig = require('../../lib/wdio/visualRegressionConf');
+const ServeStaticService = require('../../lib/wdio/services/index').ServeStaticService;
+const path = require('path');
+
+const webpackPort = 8080;
+
+exports.config = {
+  specs: [
+    path.join('tests', 'wdio', '**', '*-spec.js'),
+    path.join('packages', '*', 'tests', 'wdio', '**', '*-spec.js'),
```

Comment:
Its been noted some need to provide custom spec due to tests living in a `test` dir instead of a `tests` dir

Reply:
Lets do that in another PR, can be added passively post this release.